### ISSUE TITLE
MaskedInput - options accessibility

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7076]
As a keyboard/screenreader user it is difficult to interact with the options in MaskedInput.

- Options should be selectable with 'Enter' or 'Space'.
- There should be an indication that a drop is opened when the MaskedInput receives focus.
- While using a screenreader, after moving down into to the options list there is not a way to get back to the input.

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/input-maskedinput-time--time

### Steps to Reproduce

Try using a screenreader to interact with the story above.

**Comment by KushalNerella07** [comment]
Hi, I'd like to address the accessibility issue with MaskedInput in the storybook link provided, ensuring options are keyboard/screenreader-friendly. Can I take this on? Thanks!

**Comment by jcfilben** [comment]
Hi @KushalNerella07 appreciate your interest in working on this, feel free to create a pull request for it!

**Comment by halocline** [comment]
@jcfilben - Is this still an issue or has it been addressed?

**Comment by jcfilben** [comment]
I believe this has been resolved. Related PR: https://github.com/grommet/grommet/pull/7588.
Marking this as closed
